.buildkite/commands/dependency-diff.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -u
+
+echo "--- :memo: Commenting on the PR with the dependency diff"
+comment_with_dependency_diff 'app' 'releaseRuntimeClasspath'
```

Comment:
Question (❓): Comparing that to the other projects, why did you decide to create a separate command for dependency diff, that is, instead of using `comment_with_dependency_diff 'app' 'releaseRuntimeClasspath'` directly within the `pipeline.yml` step? 🤔

Reply:
Awesome, thanks! 👍